components/progressbar/index.css

```diff
@@ -133,14 +133,14 @@
 		block-size: var(--mod-progressbar-thickness, var(--spectrum-progressbar-thickness));
 		border-radius: var(--spectrum-progressbar-corner-radius);
 
-		background-color: var(--highcontrast-progressbar-track-color, var(--mod-progressbar-track-color, var(--spectrum-progressbar-track-color)));
+		background: var(--highcontrast-progressbar-track-color, var(--mod-progressbar-track-color, var(--spectrum-progressbar-track-color)));
```

Comment:
This was a fix we'd made in #2929 to allow gradients. I think those would only occur in the express theme so that may not be something we even need to have working in this branch, but keeping it consistent with the other branches seemed like the most reasonable thing to do, open to other opinions though!